src/pages/settings/gateways/common/components/GatewaysTable.tsx

```diff
@@ -167,7 +167,7 @@ export function GatewaysTable(params: Params) {
 
   useEffect(() => {
     if (updateCompany) {
-      handleCompanySave(updateCompany);
+      handleCompanySave({ excludeToasters: true });
```

Comment:
I wonder in which cases we want toasters? It seems like we're disabling tons of them, I wonder if opposite should be default?

Reply:
@beganovich So, we are disabling the toaster for the company endpoint in cases when another "priority" endpoint is called on that page. This priority endpoint already includes great UX with its own toaster, so having the company endpoint toaster would potentially break the UX. In this specific case that you commented on, we're disabling the toaster when users drag and drop (dnd) gateways (prioritizing payments), so users can simply drag and drop while the details are saved in the background.